joomla/joomla-cms
# Vote button missing default styling

Pull Request for Issue # .
#### Summary of Changes

added btn-default to the "rate" button to provide default bootstrap styling in 3.5.1
#### Testing Instructions

create a article that has voting turned on.
inspect the "rate" button.  You'll see that there is no default styling.
![image](https://cloud.githubusercontent.com/assets/1850089/14591355/9aaa4ec8-04d5-11e6-94d4-de5df3c01c98.png)

install the patch and you will see the styling added... and button will take on the default color style for 
buttons.
![image](https://cloud.githubusercontent.com/assets/1850089/14591361/b9b19146-04d5-11e6-8f18-840480491df9.png)
